hackathon/blender/README.md

```diff
@@ -12,34 +12,34 @@ of MD trajectories. Under the hood, *Molecular Nodes* uses MDAnalysis for trajec
 ## Background
 
 Blender is used for a wide variety of 3D modelling and animation tasks throughout the games,
-TV, movies and VFX industries. It has become a favourite among many for its speed of
-development, wide range of features and that it is 100% free and open source.
+TV, movies, and VFX industries. It has become a favourite among many for its speed of
+development, wide range of features, and that it is 100% free and open source.
 
-Blender comes bundled with it's own python kernel, which enables extremely extensive
+Blender comes bundled with its own python kernel, which enables extremely extensive
 scripting possibilities through the quite comprehensive API. This has enabled a variety of
 scientific visualisation applications that interface with, or are built on top of Blender.
 
-One such project is Molecular Nodes, which enables efficient import of structural biology
+One such project is Molecular Nodes, which enables efficient importing of structural biology
 data into Blender for advanced visualisation and rendering capabilities. For the import of
 molecular dynamics trajectories, `MDAnalysis` has been used as the interface enabling import
-in to Blender.
+into Blender.
 
-The current implementations reads all of the requested frames of the trajectory in to
-memory, and discards the MDAnalysis session after import is complete.
+The current implementations read all of the requested frames of the trajectory into
+memory, and discard the MDAnalysis session after importing is complete.
 
 ### A live MDAnalysis session
```

Comment:
Did you want to put a section about interacting over the hackathon discord somewhere?